functions/src/utils/calculateResult.js

```diff
@@ -0,0 +1,23 @@
+const calculateResult = (number1, number2, operation) => {
```

Comment:
Recently we have started to use `function` unless `const` is for a reason since then we can get scope hoisting (for if utils in the same file use each other, order then does not matter). So:

```js
export default function calculateResult(number1, number2, operation) {
```